ccid_list

```diff
@@ -0,0 +1,2 @@
+zicun
+nadi
```

Comment:
Since the spreadsheets we have all have CCIDS in the form of emails, and that's probably what you will all copy off of, does it make sense to have this in the form of emails? See [https://docs.google.com/spreadsheets/d/1xikVQ3TbYD5pg1fT7M-n_oPifiI0AVBuNrDIIwRx1DQ/edit#gid=0](https://docs.google.com/spreadsheets/d/1xikVQ3TbYD5pg1fT7M-n_oPifiI0AVBuNrDIIwRx1DQ/edit#gid=0) and [https://docs.google.com/spreadsheets/d/1ccueXQgTkS_DWQttuVNnd1wJ1-K7Mp_oFlIvTq5gnrw/edit#gid=0](https://docs.google.com/spreadsheets/d/1ccueXQgTkS_DWQttuVNnd1wJ1-K7Mp_oFlIvTq5gnrw/edit#gid=0)